api/python/quilt3/packages.py

```diff
@@ -1100,6 +1100,34 @@ def physical_key_is_temp_file(pk):
         pkg.build(name, registry=registry, message=message)
         return pkg
 
+    @classmethod
+    def rollback(cls, name, top_hash, registry=None):
+        """
+        Set the "latest" version to the given hash.
+
+        Args:
+            name(str): Name of package to rollback.
+            top_hash(str): Hash to rollback to.
+            registry(str): Registry where package is located.
+                Defaults to the default local registry.
+        """
+        if registry is None:
+            registry = get_from_config('default_local_registry')
```

Comment:
Question: do we default to local or remote?

Reply:
pretty disruptive operation so no default at all and make `registry` required?